src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,227 @@
+    </do-dont-example>
+</grid-wrapper>
+
+
+![Light theme sample](images/Light_theme_sample_revised.gif)
+
+
+_Components with common colors across both light UI backgrounds_
+
+
+### Dark theme
+
+Gray 90 components are generally paired with Gray 100 backgrounds and Gray 80 components with Gray 90 backgrounds. Some components, such as buttons and toggles, are common across all three dark UI background.
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="Gray 90 Dropdown on Gray 100 background.">
+        <img src="images/Dark_theme_01_new.png" alt="Gray 10 Dropdown on White background.">    
+    </do-dont-example>
+    <do-dont-example correct=true label="Gray 80 Dropdown on Gray 90 background.">
+        <img src="images/Dark_theme_02_rev.png" alt="White Dropdown on a Gray 10 background.">    
+    </do-dont-example>
+</grid-wrapper>
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="Gray 70 is sometimes used when compound components are being applied such as an overflow menu.">
+        <img src="images/Dark_theme_03_new.png" alt="White Overflow Menu on Gray 10 card.">    
+    </do-dont-example>
+    <do-dont-example label="Do not apply components that are darker than the background unless using high contrast mode.">
+        <img src="images/Dark_theme_01_new.png" alt="Do not apply components that are darker than the background unless using high contrast mode.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+![Dark theme sample](images/Dark_theme_sample_revised.gif)
+
+_Components with common colors across both dark UI backgrounds_
+
+### High contrast
+
+In some cases, it is helpful to apply light components to dark backgrounds or light components to dark backgrounds. This technique is useful to focus attention or create visual tension when necessary.
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label=" ">
+        <img src="images/High_contrast_01.png" alt="High contrast example in a light theme.">    
+    </do-dont-example>
+    <do-dont-example correct=true label=" ">
+        <img src="images/High_contrast_02.png" alt="High contrast example in a dark theme.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+## Interaction states
+
+In addition to the enabled state tokens above, the five other primary interaction states have also been defined and tokenized for each theme. For values between Black and 70, interaction gets lighter and for values between 60 and White, interaction gets darker.
+
+#### Hover
+A unique set of hex values and color tokens exists for the hover states. These colors are outside of the core palette steps and can be considered as half steps in between two adjacent colors. In order to simplify our token structure, the White theme shares the same hover value as the Gray 10 theme; the two dark theme backgrounds share a hover value as well.
```

Comment:
"These colors are outside of the core **IBM Design Language color** palette steps"

Reply:
"In order to simplify our token structure, the White theme shares the same hover value as the Gray 10 theme; the two dark theme backgrounds share a hover value as well." -- again slightly not true to how its being implemented. They don't share the same tokens they just both get the same assigned color values.

Rewrite to something like:
"The two light themes share the same color values for each interaction tokens; the two dark themes share their same color values as well."

Reply:
small adjustment: "The two light themes share the same color values for each interaction token; the two dark themes follow the same rule."